ci/infra/vmware/master-instance.tf

```diff
@@ -118,4 +124,12 @@ resource "null_resource" "master_wait_cloudinit" {
       "cloud-init status --wait > /dev/null",
     ]
   }
+
+  provisioner "remote-exec" {
+    when       = destroy
+    on_failure = "continue"
```

Comment:
It works with `on_failure = continue`, now tf is able to destroy cluster (tried on openstack) even when one node is offline.

It looks like the remote-exec is called several times and after 5 minutes it gives up and destroys the cluster anyway.

 But please remove quotes around `continue` as these are deprecated for keywords now:
```
Warning: Quoted keywords are deprecated

  on master-instance.tf line 125, in resource "null_resource" "master":
 125:     on_failure = "continue"

In this context, keywords are expected literally rather than in quotes.
Terraform 0.11 and earlier required quotes, but quoted keywords are now
deprecated and will be removed in a future version of Terraform. Remove the
quotes surrounding this keyword to silence this warning.
```